doc/lua_api.txt

```diff
@@ -3364,6 +3364,23 @@ It can be created via `Settings(filename)`.
     * write changes to file
 * `to_table()`: returns `{[key1]=value1,...}`
 
+### `Raycast`
```

Comment:
Yes, I did. But since `Raycast` can be used in a `for` loop, it is not necessary to have `start_raycast()` and `raycast_iter()`.